Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -1669,6 +1744,61 @@ void CMultiplayerSA::SetColorFilter(DWORD dwPass0Color, DWORD dwPass1Color)
     }
 }
 
+void CMultiplayerSA::SetGrainMultiplier(eGrainMultiplierType type, float fMultiplier)
```

Comment:
Now that I think about it, if you really wanted to complicate things, you could make a class called `Modifier`, with a template argument `eGrainMultiplierType`.
For ex, nigh vision would become: `Modifier<eGrainMultiplierType::NIGHT>`.

Just a thought though.